webpack.config.js

```diff
@@ -513,15 +513,28 @@ async function initializeModuleFederation() {
 
             // Other containers will be forced to use the exact versions of shared modules that the web app provides.
             makeSingletonSharedModules([
+                'history',
                 'react',
+                'react-beautiful-dnd',
                 'react-bootstrap',
                 'react-dom',
                 'react-intl',
                 'react-redux',
                 'react-router-dom',
             ]),
         ],
-    }));
+    };
+
+    // Desktop specific code for remote module loading
+    moduleFederationPluginOptions.exposes = {
+        './app': 'components/app.jsx',
+        './store': 'stores/redux_store.jsx',
+        './styles': 'sass/styles.scss',
+        './registry': 'module_registry',
```

Comment:
just checking do we not `ts` or webpack will resolve it to correctly?

Reply:
Webpack seems to resolve this correctly. I think it's because it's in the root folder.